docs/configuration/configuration.md

```diff
@@ -529,6 +529,11 @@ region: <string>
 [ project_name: <string> ]
 [ project_id: <string> ]
 
+# If you need to discover all instances from openstack (project wide) change
+# all_tenants to true. Otherwise it will only discover instances from the
+# specified project_name or project_id even if you provide admin credentials.
```

Comment:
Suggestion based on the [Compute API v2](https://developer.openstack.org/api-ref/compute/#list-servers-detailed) documentation:

```
Whether the service discovery should list all instances for all projects. It is only relevant for the 'instance' role and usually requires admin permissions.
```

Reply:
Done.

Reply:
You can remove L532-534 though